CoreOpenGLDisplay/src/au/gov/asd/tac/constellation/visual/opengl/utilities/SharedDrawable.java

```diff
@@ -129,28 +122,39 @@ private static void init() {
         }
     }
 
-    public static void exportGlyphTextures(File baseFile) {
+    public static void exportGlyphTextures(final File baseFile) {
 
         // Ensure that JavaFX is running
-        Platform.startup(() -> {
-        });
-
-        Platform.runLater(() -> {
-            String baseFileName = baseFile.getAbsolutePath();
-            int extensionStart = baseFileName.lastIndexOf('.');
-            if (extensionStart > 0) {
-                baseFileName = baseFileName.substring(0, extensionStart);
-            }
+        try {
+            Platform.startup(() -> {
+            });
+        } catch (final IllegalStateException ex) {
+            /**
+             * there isn't a way to tell whether the JavaFX platform is running
+             * so we'll absorb this exception and move on.
+             */
+        }
 
-            for (int page = 0; page < glyphManager.getGlyphPageCount(); page++) {
-                File outputFile = new File(baseFileName + SeparatorConstants.UNDERSCORE + page + ".png");
-                try (OutputStream out = new FileOutputStream(outputFile)) {
-                    glyphManager.writeGlyphBuffer(page, out);
-                } catch (IOException ex) {
-                    // Nothing to do here - this is just for developer testing.
+        if (glyphManager != null) {
+            Platform.runLater(() -> {
+                String baseFileName = baseFile.getAbsolutePath();
+                final int extensionStart = baseFileName.lastIndexOf('.');
+                if (extensionStart > 0) {
+                    baseFileName = baseFileName.substring(0, extensionStart);
                 }
-            }
-        });
+
+                for (int page = 0; page < glyphManager.getGlyphPageCount(); page++) {
+                    final File outputFile = new File(baseFileName + SeparatorConstants.UNDERSCORE + page + ".png");
+                    try ( OutputStream out = new FileOutputStream(outputFile)) {
+                        glyphManager.writeGlyphBuffer(page, out);
+                    } catch (IOException ex) {
+                        // Nothing to do here - this is just for developer testing.
```

Comment:
is the intention to remove this once developer testing is done?

Reply:
I'd like to keep this plugin around - so I've replaced the comment with a `logger` call.